src/UXClient/Components/Slider/Slider.scss

```diff
@@ -8,6 +8,9 @@
    
     .tsi-sliderTrack {
         stroke: $gray1;
+        @media screen and (-ms-high-contrast: white-on-black) { 
+            stroke: white !important;          
```

Comment:
Is this !important flag needed here? Since for the non-high-contrast stroke it isn't needed?